app/assets/stylesheets/library_hours.scss

```diff
@@ -12,15 +12,20 @@
   @include media-breakpoint-up(sm) {
     .date-input {
       padding-right: ($spacer * .75);
+      .form-control:focus {
```

Comment:
Is there are reason we only want this in certain viewport widths?

Reply:
Thank you, Justin. This is an error, likely when I was consolidating my sizes. I found another error of the same type as well.